### ISSUE TITLE
Full-page redirect is not working with the new Shopify admins URL

### ISSUE THREAD

**Issue by gnumarcelo** [kirillplatonov/shopify-hotwire-sample#72]
Since Shopify started changing Store's admin domains from `https://my-awesome-store.myshopify.com/admin` to `https://admin.shopify.com/store/my-awesome-store` it looks like the `fullpage_redirect_to` is not working anymore, unfortunately  without any error/log message being printed.

I'm not sure this is som

Steps to replicate it:
1 - Boot the current app
2 - Install it into your store (making sure your store uses the new domain scheme)
3 - Click on `Fullpage Redirect` or `Remote Redirect` and see that nothing happens

**Comment by arjunrajkumar** [comment]
Hi @gnumarcelo .. just checking if you were able to get the fullpage_redirect_to working? Thinking this will be the same while redirecting to the subscription confirmation_url also?

**Comment by kirillplatonov** [comment]
Yeah, they're completely broken in the ShopifyApp gem right now. It's time implement full page redirects directly in the sample app 🤦‍♂️
